### ISSUE TITLE
media-feature-name-case

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#1031]
Here's another one for [media feature names](https://www.w3.org/TR/mediaqueries-4/#mq-features)

```css
/* media-feature-name-case: "upper" */
@media (MIN-WIDTH: 10px) {}
```

- Name: `media-feature-name-case`
- Primary options: `"upper"|"lower"`
- Secondary options: none
- Message: `Expected "${actual}" to be "${expected}" (media-feature-name-case)`.

I can't think of any more off hand. Create a new issue if anyone else can think of any?

**Comment by jeddy3** [comment]
This will likely need the creation of a `postcss-media-parser` module. Ref: https://github.com/stylelint/stylelint/issues/1033

**Comment by evilebottnawi** [comment]
@jeddy3 @davidtheclark i think we should create `postcss-media-parser` before release v7.0.0, because, rules `unit-*` should see on value on media feature (10px, 10 dpi and etc), this is breaking changes.

**Comment by jeddy3** [comment]
> rules `unit-*` should see on value on media feature (10px, 10 dpi and etc), this is breaking changes.

Yep, that's a good point. I've created a [separate issue](https://github.com/stylelint/stylelint/issues/1505) for this.

> i think we should create `postcss-media-parser`

You're closest to these `media-feature-*` rules than anyone else and if you think `postcss-value-parser` doesn't offer the fidelity this rule needs, then I say go ahead and create a `postcss-media-parser` module.

**Comment by jeddy3** [comment]
@evilebottnawi FYI, https://github.com/dryoma/postcss-media-query-parser

**Comment by evilebottnawi** [comment]
@jeddy3 cool, i wait release v7 :smile_cat: and lets new rules

**Comment by jeddy3** [comment]
> and lets new rules

As long as it's at a sustainable pace :) I think we could all do with a little breather after all the hard work put into the `7.0.0` release.

**Comment by evilebottnawi** [comment]
@jeddy3 `eat sleep code repeat` :racehorse:

**Comment by jeddy3** [comment]
Done in #1660
